bindings/wasm/src/author/authorw.rs

```diff
@@ -332,4 +344,28 @@ impl Author {
             })
             .into_js_result()
     }
+
+    #[wasm_bindgen(catch)]
+    pub fn store_new_subscriber(&self, pk_str: String) -> Result<()> {
+        public_key_from_string(&pk_str).map_or_else(
+            |err| Err(err),
+            |pk| self.author.borrow_mut().store_new_subscriber(pk).into_js_result(),
+        )
```

Comment:
A more idiomatic way to write the same:
```suggestion
        public_key_from_string(&pk_str)
            .and_then(|pk| self.author.borrow_mut().store_new_subscriber(pk))
            .into_js_result()
```

Reply:
This applies also to the other similar methods

Reply:
I've updated them